gst/nnstreamer/include/nnstreamer_plugin_api_converter.h

```diff
@@ -26,9 +26,11 @@
 
 #include "tensor_typedef.h"
 #include <gst/gst.h>
+#include <tensor_converter/tensor_converter.h>
```

Comment:
tensor_convert.h is not to be exposed to other packages.
Which symbol are you going to refer from it?

Reply:
Thank you. I reverted it.